concepticondata/conceptlists/Johansson-2020-344.tsv

```diff
@@ -0,0 +1,345 @@
+Johansson-2020-344-84	84	FOOT	1301	FOOT
+Johansson-2020-344-85	85	FOUR	1500	FOUR
+Johansson-2020-344-86	86	FULL	1429	FULL
+Johansson-2020-344-87	87	GIRL	1646	GIRL
+Johansson-2020-344-88	88	GIVE	1447	GIVE
+Johansson-2020-344-89	89	GO	695	GO
+Johansson-2020-344-90	90	GOOD	1035	GOOD
+Johansson-2020-344-91	91	GRASS	606	GRASS
+Johansson-2020-344-92	92	GRAY	153	GREY
+Johansson-2020-344-93	93	GREASE	323	FAT (ORGANIC SUBSTANCE)
+Johansson-2020-344-94	94	GREEN	1425	GREEN
+Johansson-2020-344-95	95	GROW	479	GROW
+Johansson-2020-344-96	96	HAIR	1040	HAIR
+Johansson-2020-344-97	97	HALF	816	HALF
+Johansson-2020-344-98	98	HAND	1277	HAND
+Johansson-2020-344-99	99	HAPPY	1495	HAPPY
+Johansson-2020-344-100	100	HARD	1884	HARD
+Johansson-2020-344-101	101	INTERCOURSE		
+Johansson-2020-344-102	102	HEAD	1256	HEAD
+Johansson-2020-344-103	103	HEAR	1408	HEAR
+Johansson-2020-344-104	104	HEART	1223	HEART
+Johansson-2020-344-105	105	HEAVY	1210	HEAVY
+Johansson-2020-344-106	106	HERE	136	HERE
+Johansson-2020-344-107	107	HIDE	2486	HIDE
+Johansson-2020-344-108	108	HIGH	1265	HIGH
+Johansson-2020-344-109	109	HIT	1433	HIT
+Johansson-2020-344-110	110	HORN	1393	HORN (ANATOMY)
+Johansson-2020-344-111	111	HOT	1286	HOT
+Johansson-2020-344-112	112	HOUSE	1252	HOUSE
+Johansson-2020-344-113	113	HOW?	1239	HOW
+Johansson-2020-344-114	114	IF	1459	IF
+Johansson-2020-344-115	115	IN FRONT OF	1883	IN FRONT OF
+Johansson-2020-344-116	116	IN(SIDE)	1460	IN
+Johansson-2020-344-117	117	KILL	1417	KILL
+Johansson-2020-344-118	118	KNEE	1371	KNEE
+Johansson-2020-344-119	119	KNOW	3626	KNOW
+Johansson-2020-344-120	120	LAUGH	1355	LAUGH
+Johansson-2020-344-121	121	LEAF	628	LEAF
+Johansson-2020-344-122	122	LEFT	244	LEFT
+Johansson-2020-344-123	123	LOWER LEG	447	LOWER LEG
+Johansson-2020-344-124	124	LIE (DOWN)	215	LIE DOWN
+Johansson-2020-344-125	125	LIGHT (NOT DARK)	250	LIGHT (COLOR)
+Johansson-2020-344-126	126	LIGHT (WGT)	1052	LIGHT (WEIGHT)
+Johansson-2020-344-127	127	LIP	478	LIP
+Johansson-2020-344-128	128	LIVE	1422	BE ALIVE
+Johansson-2020-344-129	129	LIVER	1224	LIVER
+Johansson-2020-344-130	130	LONG	1203	LONG
+Johansson-2020-344-131	131	LOUD	377	LOUD
+Johansson-2020-344-132	132	LOUSE	1392	LOUSE
+Johansson-2020-344-133	133	LOW	721	LOW
+Johansson-2020-344-134	134	LUNG	688	LUNG
+Johansson-2020-344-135	135	MAN	1554	MAN
+Johansson-2020-344-136	136	MANY	1198	MANY
+Johansson-2020-344-137	137	MAYBE		
+Johansson-2020-344-138	138	MILK	635	MILK
+Johansson-2020-344-139	139	MOON	1313	MOON
+Johansson-2020-344-140	140	MOUNTAIN	639	MOUNTAIN
+Johansson-2020-344-141	141	MOUTH	674	MOUTH
+Johansson-2020-344-142	142	NAME	1405	NAME
+Johansson-2020-344-143	143	NARROW	1267	NARROW
+Johansson-2020-344-144	144	NAVEL	1838	NAVEL
+Johansson-2020-344-145	145	NEAR	1942	NEAR
+Johansson-2020-344-146	146	NECK	1333	NECK
+Johansson-2020-344-147	147	NEW	1231	NEW
+Johansson-2020-344-148	148	NIGHT	1233	NIGHT
+Johansson-2020-344-149	149	NINE	1483	NINE
+Johansson-2020-344-150	150	NIPPLE	796	NIPPLE
+Johansson-2020-344-151	151	NO(THING)	687	NOTHING
```

Comment:
Perhaps 3198 NOTHING OR ZERO to account for the "no"?

Reply:
NOTHING OR ZERO seems too specific for this case as it necessarily involves the cardinal number. In Johansson study this concept is also spelled as 'NOTHING' and is referred to by the author as a negatory quantifier.